specification/purview/Azure.Analytics.Purview.DataMap/routes.tsp

```diff
@@ -0,0 +1,2245 @@
+  @doc("Get the struct definition for the given GUID.")
+  @route("/atlas/v2/types/structdef/guid/{guid}")
+  @get
+  getStructDefByGuid is Azure.Core.Foundations.Operation<
+    {
+      @doc("The globally unique identifier of the struct.")
+      @path
+      guid: string;
+    },
+    AtlasStructDef,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("Get the struct definition by its name (unique).")
+  @route("/atlas/v2/types/structdef/name/{name}")
+  @get
+  getStructDefByName is Azure.Core.Foundations.Operation<
+    {
+      @doc("The name of the struct.")
+      @path
+      name: string;
+    },
+    AtlasStructDef,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("Get the type definition for the given GUID.")
+  @route("/atlas/v2/types/typedef/guid/{guid}")
+  @get
+  getByGuid is Azure.Core.Foundations.Operation<
+    {
+      @doc("The globally unique identifier of the type.")
+      @path
+      guid: string;
+    },
+    AtlasTypeDef,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("Get the type definition by its name (unique).")
+  @route("/atlas/v2/types/typedef/name/{name}")
+  @get
+  getByName is Azure.Core.Foundations.Operation<
+    {
+      @doc("The name of the type.")
+      @path
+      name: string;
+    },
+    AtlasTypeDef,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("Delete API for type identified by its name.")
+  @route("/atlas/v2/types/typedef/name/{name}")
+  @delete
+  delete is Azure.Core.Foundations.Operation<
+    {
+      @doc("The name of the type.")
+      @path
+      name: string;
+    },
+    void,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("List all type definitions in bulk.")
+  @route("/atlas/v2/types/typedefs")
+  @get
+  list is Azure.Core.Foundations.Operation<
+    {
+      @doc("""
+Whether include termtemplatedef when return all typedefs.
+This is always true
+when search filter type=term_template
+""")
+      @query
+      includeTermTemplate?: boolean;
+
+      @doc("Typedef name as search filter when get typedefs.")
+      @query
+      type?: TypeDef;
+    },
+    AtlasTypesDef,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("""
+Create all atlas type definitions in bulk, only new definitions will be
+created.
+Any changes to the existing definitions will be discarded.
+""")
+  @route("/atlas/v2/types/typedefs")
+  @post
+  bulkCreate is Azure.Core.Foundations.Operation<
```

Comment:
```
  bulkCreateTypeDefinitions is Azure.Core.Foundations.Operation<
```

Reply:
@johanste was this a suggestion for the SDK?